### PR TITLE
Compatible hackuity api token auth

### PR DESCRIPTION
**Pull request by thefrieddan1**
## Description
Deprecate login and password use api key

### CODE REVIEW

**Review by Rogdham — COMMENTED** [review]
Hello, I have tested this PR from on a fresh integration instance, and it works fine.

File: Packs/Hackuity/Integrations/Hackuity/README.md [hunk 1]
```diff
@@ -11,8 +11,8 @@ This integration was integrated and tested with version 1.25.0 of Hackuity
     | --- | --- |
     | Corporate server URL | True |
     | Namespace | True |
-    | Login | True |
-    | Password | True |
+    | Login | False |
+    | Api key | False |
```
[the remark this answers is not in the document]
Comment (thefrieddan1) [reply Packs/Hackuity/Integrations/Hackuity/README.md:15]: You are right. Thanks.

File: Packs/Hackuity/ReleaseNotes/1_0_7.md [hunk 2]
```diff
@@ -1,3 +1,3 @@
 #### Integrations
 ##### Hackuity
-- Updated the Docker image to: *demisto/python3:3.10.11.61265*.
+- Updated the Docker image to: *demisto/python3:3.10.11.61265*.
```
Comment (eyalpalo) [inline Packs/Hackuity/ReleaseNotes/1_0_7.md:3]: Why do we see this change? Please revert (maybe a new line at the end of the file is missing)

**Review by eyalpalo — APPROVED** [review]
Please see the comment about the RN before merging

File: Packs/Hackuity/ReleaseNotes/1_0_7.md (hunk 2 above)
Comment (thefrieddan1) [reply Packs/Hackuity/ReleaseNotes/1_0_7.md]: Done, indeed a new line in the end of file.

**Pull request merged**